nginx-snippet.conf

```diff
@@ -1,5 +1,5 @@
 location ~ /(.*)$ {
-            location ~ ^/(.*/(config\.seed|sha256sums|sha256sums\.gpg|Packages|Packages\.gpg|Packages\.manifest|Packages\.sig))$ {
+            location ~ ^/(.*/(config\.seed|sha256sums|sha256sums\.gpg|Packages|Packages\.gpg|Packages\.manifest|Packages\.sig|*.buildinfo))$ {
```

Comment:
I don't think this pattern will work. You probably meant something like `.+\.buildinfo`

Reply:
Fixed, thanks you